### DISCUSSION TITLE
Add option to change release title

### DISCUSSION THREAD

**Discussion by Katsute** [community/community#6020] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like the option to change the release notes title to keep it in line with my existing release notes.

![gh-release-title](https://user-images.githubusercontent.com/58778985/136108752-5e9337e9-49c4-4c45-9f5a-c6326655b254.PNG)

**Comment by MylesBorins** [comment, marked as answer]
@Katsute we explored this as an option but supporting this feature would require introducing variables into the configuration which was something that was out of scope for this iteration. The good news is that we have built the infrastructure in such a way that we could support this in the future!
